helm_deploy/values-dev.yaml

```diff
@@ -8,6 +8,7 @@ generic-service:
     host: book-a-video-link-api-dev.prison.service.justice.gov.uk
 
   env:
+    SPRING_PROFILES_ACTIVE: dev
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
```

Comment:
Does it need this? The spring profile dev is usually for running locally, isn't it?

Reply:
There is a local profile for running locally.

There is the dev profile added as part of this PR so we can override values the default application.yml file.

This is telling Spring to override the main profile with the dev profile file when deploying to dev.